merlin/models/tf/core/combinators.py

```diff
@@ -239,9 +239,9 @@ def trainable(self):
         Returns
         -------
         bool
-            True if all layer within SequentialBlock are trainable, otherwise False
+            True if any layer within SequentialBlock are trainable, otherwise False
         """
-        return all(layer.trainable for layer in self.layers)
+        return any(layer.trainable for layer in self.layers)
```

Comment:
What the reasoning behind this change? `trainable` is a standard keras-method, and if it returns `any` I am worried it will have some unforeseen side-effects.

Reply:
There are many `ParallelBlock`s in a model, and sometime we would freeze some children blocks of it, so parts of its children blocks have `trainable=True` and other children blocks have `trainable=False`, if we keep `all` here, then the entire `ParallelBlock` would have `trainable` as False, it is not we want, we still want to change the unfrozen children blocks.